spec/controllers/submissions_controller_spec.rb

```diff
@@ -19,4 +19,47 @@
         .to change { Submission.where(submission_attrs).exists? }.to(true)
     end
   end
+
+  describe '#approve' do
+    let!(:pending_submission) { Submission.create!(body: 'Pending submission') }
+    let(:params) { { submission_id: pending_submission.id } }
+
+    context 'with admin authentication' do
+      let(:admin) do
+        Admin.create!(email: 'admin@example.com', password: 'Password1')
+      end
```

Comment:
```suggestion
      }
```

Reply:
Thanks @matt-wratt, I missed this one. All fixed.